src/styles.scss

```diff
@@ -144,9 +153,10 @@ body {
     -ms-user-select: none;
     outline: none;
 
-    &:hover, &:focus {
+    &:focus,
+    &:hover {
       background-color: rgba(0, 0, 0, 0.2);
-      opacity: .95;
+      opacity: 0.95;
       color: white;
       user-drag: none;
```

Comment:
Dont think so. That css was copied from our styles guideline for buttons. I will remove the property.